internal/host/hostcommands/install_cmd.go

```diff
@@ -288,10 +288,16 @@ func appendDHCPArgs(cluster *common.Cluster, installerArgs []string, log logrus.
 		return append(installerArgs, "--append-karg", "ip=dhcp"), nil
 	}
 
-	if swag.StringValue(cluster.Kind) == models.ClusterKindAddHostsCluster {
-		return installerArgs, nil
+	if swag.StringValue(cluster.Kind) != models.ClusterKindAddHostsCluster {
+		return installerArgs, errors.Errorf("cannot determine machine network address family")
+	}
+
+	if v4, v6, err := hostutil.GetAddressFamilies(host); err != nil {
+		return installerArgs, err
```

Comment:
don't we want to return error in case unmarshall on host inventory failed?

Reply:
We do `; err != nil {`

Reply:
right, my mistake